src/Umbraco.Web/Security/MembershipHelper.cs

```diff
@@ -19,6 +19,15 @@
 namespace Umbraco.Web.Security
 {
     // MIGRATED TO NETCORE
+    // TODO: Analyse all - much can be moved/removed since most methods will occur on the manager via identity implementation
```

Comment:
`MembershipHelper.cs` to be removed once all methods have been analysed, moved/removed since most methods will occur on the manager via identity implementation